platform/Dockerfiles/rust-hypervisor-firmware/build.sh

```diff
@@ -0,0 +1,7 @@
+RUST_HYPERVISOR_FIRMWARE_VERSION=v0.4.2
+RUST_HYPERVISOR_FIRMWARE_EPOCH=0
+date=$(date '+%Y%m%d%H%M%S')
+
+docker buildx build --tag artificialwisdomai/rust-hypervisor-version:${RUST_HYPERVISOR_FIRMWARE_VERSION}-${RUST_HYPERVISOR_FIRMWARE_EPOCH}-${date} --output type=docker --progress plain .
```

Comment:
`rust-hypervisor-version` should be `rust-hypervisor-firmware`. I will shortly push an update.